### PR TITLE
Refacor/ASC-558 convert tsx to mdx

### PR DESCRIPTION
**Pull request by jasperswart**
sorry for the size of this pr

In this PR:
- refactored all remaining components stories to mdx type 
- only 1 component story remains: SearchBarToggle

### CODE REVIEW

File: stories/src/ui/FilterTag.stories.mdx [hunk 1]
```diff
@@ -14,7 +15,7 @@ import { FilterTag } from '@amsterdam/asc-ui'
 <Preview>
   <Story name="Default">
     <React.Fragment>
-      <FilterTag onClick={() => alert('Remove me!')}>Just a tag</FilterTag>
+      <FilterTag onClick={(e) => action('Remove me!')(e)}>Just a tag</FilterTag>
```
Comment (robinpiets) [inline stories/src/ui/FilterTag.stories.mdx:18]: This action doesn't work

File: stories/src/ui/ImageCard.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Doesn't exist in Design System yet
+
+**Component status: Experimental**
+
+## Default
+
+<Preview>
+  <Story name="Default">
+    <ImageCard backgroundImage="http://lorempixel.com/output/food-q-c-640-480-3.jpg">
```
Comment (robinpiets) [inline stories/src/ui/ImageCard.stories.mdx:21]: This one looks massive. In the old version it looked a lot smaller. Maybe you can wrap it a div with only 50% width or something

**Review by robinpiets — CHANGES_REQUESTED** [review]
Looks good. Just some minor things. And can you make change in the changelog?

File: stories/src/ui/FilterTag.stories.mdx (hunk 1 above)
Comment (robinpiets) [reply stories/src/ui/FilterTag.stories.mdx:18]: Oh it does work actually... my bad

File: stories/src/ui/ImageCard.stories.mdx (hunk 2 above)
Comment (jasperswart) [reply stories/src/ui/ImageCard.stories.mdx:21]: ja dat is een goed punt: ik ga even de max-widh op alle componenten zetten

Comment (jasperswart) [reply stories/src/ui/ImageCard.stories.mdx:35]: gedaan

**Review by jonkoops — APPROVED** [review]

**Pull request merged**